docs/tempo/website/configuration/tls.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Configure TLS Communication
+weight: 70
+---
+
+# Configure TLS Communication
```

Comment:
I changed communication to lower case since we primarily use sentence case for titles. Unless TLS Communication is a proper noun? 

```suggestion
# Configure TLS communication
```